### PR TITLE
Support more columns in migrate function.

### PR DESCRIPTION
**Pull request by eckelarsson**
Simple switch case to check what type of column to create depending on the value given.
The main reason was to support array values.

### CODE REVIEW

File: src/Sushi.php [hunk 1]
```diff
@@ -88,8 +88,24 @@ public function migrate()
             }
 
             foreach ($firstRow as $column => $value) {
-                $type = is_numeric($value) ? 'integer' : 'string';
-
+                 
+                switch (true) {
+                    case is_numeric($value):
+                        $type = 'integer';
```
Comment (Gummibeer) [inline src/Sushi.php]: `is_numeric()` returns also `true` for floats.
https://www.php.net/manual/en/function.is-numeric.php

```suggestion
                    case is_int($value):
                        $type = 'integer';
                        break;
                    case is_numeric($value):
                        $type = 'float';
```

File: src/Sushi.php [hunk 2]
```diff
@@ -88,8 +88,24 @@ public function migrate()
...
+                    case is_array($value):
+                        $type = 'longtext';
+                        break;
+                    case is_float($value):
+                        $type = 'float';
+                        break;
+                    case is_string($value):
+                        $type = 'string';
+                        break;
+                    default:
```
Comment (Gummibeer) [inline src/Sushi.php]: I think that adding support for date times would be nice.

```suggestion
                    case is_object($value) && $value instanceof \DateTime:
                        $type = 'timestamp';
                        break;
                    default:
```

**Comment by Gummibeer** [comment]
Relates to https://github.com/calebporzio/sushi/pull/7#issuecomment-580327975
@eckelarsson thanks for adding it. :)

**Comment by calebporzio** [comment]
I like it, I also like @Gummibeer's suggestion.

**Pull request merged**